packages/usa-hero/src/styles/_usa-hero.scss

```diff
@@ -30,6 +30,10 @@
   @include typeset-h2;
   color: color("accent-cool");
   line-height: line-height("heading", 2);
+
+  * + & {
+    margin-top: $theme-heading-margin-top;
+  }
```

Comment:
Happy to remove this with your approval. In my opinion it looks better without _but_ this was only set there to keep the styles consistent with the develop css output.

Thoughts?

Reply:
Okay, I see this does compile on `develop`.

Reply:
Good point, @mejiaj. I was only testing for consistency with the CSS output in `develop`, but didn't actually check if it is actually beneficial to have there in the first place. This one feels like it would be an improvement to remove it.